airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSourceOperations.java

```diff
@@ -128,7 +129,7 @@ protected void putBinary(final ObjectNode node,
                            final int index)
       throws SQLException {
     final byte[] bytes = resultSet.getBytes(index);
-    final String value = new String(bytes, Charset.defaultCharset());
+    final String value = Base64.encodeBase64String(bytes);
```

Comment:
this I think is in line with what we do in other java connectors 👍

Reply:
You can add a test also for binary type

Reply:
Does that mean that a binary field in the source database is encoded as a AirbyteType String? Is that something that was discussed, or just how it was implemented for simplicity reasons?

Reply:
the current behavior for binary column is undefined in our public doc (and see slack discussion with destination team here https://airbytehq-team.slack.com/archives/C03C4AVJWG4/p1700173565688209), it sounds like base64 is the direction we are heading to; so yes mostly for simplicity reason.